server/chat-commands/room-sections.ts

```diff
@@ -0,0 +1,99 @@
+import {Utils} from '../../lib';
```

Comment:
Does this really need its own chat plugin? Sim authority/recognition generally goes in in `moderation.ts`.